tools/generators/legacy/src/Generator/CreateAutogeneratedXCSchemes.swift

```diff
@@ -120,7 +121,8 @@ extension Generator {
                         buildMode: buildMode,
                         schemeInfo: schemeInfo
                     ),
-                    isTopLevel: pbxTarget.isTopLevel
+                    isTopLevel: pbxTarget.isTopLevel,
+                    isApplication: pbxTarget.productType == PBXProductType.application
```

Comment:
nit: Make extension like `pbxTarget.isTopLevel`

Reply:
Thanks for the suggestion! [Moved logic to extension file.](https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2211/files#diff-176ddd00ccbdfa85509edd61a4e12ad0ca0d6d51e6b16acb57f2753a3cea2618R284-R328)